pages/linux/xfreerdp.md

```diff
@@ -22,3 +22,7 @@
 - Connect to a FreeRDP server ignoring any certificate checks:
 
 `xfreerdp /v:{{ip_address}} /u:{{username}} /p:{{password}} /cert:ignore`
+
+- Connect to a FreeRDP server with shared drive mapped:
+
+`xfreerdp /v:ip_address /u:username /p:password /drive:{{path/to/directory}},{{share_name}}`
```

Comment:
```suggestion
`xfreerdp /v:{{ip_address}} /u:{{username}} /p:{{password}} /drive:{{path/to/directory}},{{share_name}}`
```
Thanks for your contribution and welcome to tldr. We use `{{}}` syntax for our clients to highlight placeholders in order to distinguish them from the rest of the command.

![image](https://github.com/tldr-pages/tldr/assets/26346867/6235271f-ef3e-4a5e-bf4e-be08d158fb48)

Reply:
Thank you. I should have included the {{}} syntax for all of the placeholders